packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -5,5 +5,54 @@
 # CWD is agoric-sdk
 here=./upgrade-test-scripts/agoric-upgrade-11
 
-# run zoe thru "null upgrade"
-$here/zoe-upgrade/zoe-upgrade-driver.sh
```

Comment:
If we're deleting this, let's delete `zoe-upgrade/zoe-upgrade-driver.sh` too. Perhaps the whole directory (after moving the kernel query tool).